frontend/src/components/form/Calendar.vue

```diff
@@ -40,6 +48,8 @@ const { errorMessage, value } = useField<string>(toRef(props, 'name'));
       date-format="yy/mm/dd"
     />
     <small :id="`${name}-help`">{{ helpText }}</small>
-    <ErrorMessage :name="name" />
+    <div>
+      <ErrorMessage :name="name" />
+    </div>
```

Comment:
Is the new div wrapper addressing some kind of strange spacing/layout issue? Usually we try to avoid nesting more DOM elements whenever possible.